app/routes/signup.tsx

```diff
@@ -135,61 +142,198 @@ export const loader: LoaderFunction = async ({request}) => {
   })
 }
 
+const TEAM_MAP = {
+  BLUE: {
+    image: images.alexBlue,
+    label: 'Blue Team',
+    focusClassName: 'ring-team-blue',
+  },
+  RED: {
+    image: images.alexRed,
+    label: 'Red Team',
+    focusClassName: 'ring-team-red',
+  },
+  YELLOW: {
+    image: images.alexYellow,
+    label: 'Yellow Team',
+    focusClassName: 'ring-team-yellow',
+  },
+}
+
+interface TeamOptionProps {
+  team: keyof typeof TEAM_MAP
+  error?: string | null
+  selected: boolean
+}
+
+function TeamOption({team: value, error, selected}: TeamOptionProps) {
+  const team = TEAM_MAP[value]
+
+  return (
+    <div
+      className={clsx(
+        'relative col-span-full mb-3 bg-gray-100 dark:bg-gray-800 rounded-lg focus-within:outline-none ring-team-current ring-offset-4 focus-within:ring-2 lg:col-span-4 lg:mb-0',
+        team.focusClassName,
+        {
+          'ring-2 ring-offset-team-current': selected,
+          'dark:ring-offset-gray-900 ring-offset-white': !selected,
+        },
+      )}
+    >
+      {selected ? (
+        <span className="absolute left-9 top-9 text-team-current">
+          <CheckIcon />
+        </span>
+      ) : null}
+
+      <label className="block pb-12 pt-20 px-12 text-center">
+        <input
+          className="sr-only"
+          type="radio"
+          name="team"
+          value={value}
+          aria-describedby={error ? 'team-error' : undefined}
+        />
+        <img className="block mb-16" src={team.image.src} alt="" />
+        <H6>{team.label}</H6>
+      </label>
+    </div>
+  )
+}
+
 export default function NewAccount() {
   const data = useRouteData<LoaderData>()
+  const [formValues, setFormValues] = React.useState({
+    firstName: '',
+    team: '',
+  })
+
+  const formIsValid =
+    formValues.firstName.trim().length > 0 &&
+    Object.keys(TEAM_MAP).includes(formValues.team)
```

Comment:
If we [move this](https://github.com/kentcdodds/remix-kentcdodds/blob/781a1f33612f43514eb2de6f63f3ea61c6740fd0/app/utils/prisma.server.ts#L196) to `misc.tsx` (so it's not in a `.server.ts` file), then we could do this:

```suggestion
    teams.includes(formValues.team)
```

Reply:
I'm moving that now myself 👍